pyproject.toml

```diff
@@ -37,11 +37,13 @@ pydantic = "^1.9.0"
 cachelib = "^0.10.2"
 loguru = "^0.7.0"
 deepdiff = "^6.3.0"
-cognite-gql-pygen = ">= 0.4.4"
 matplotlib = "^3.7.1"
 dacite = "^1.8.1"
 python-dotenv = "^1.0.0"
 arrow = "^1.2.0"
+mergedeep = "^1.3.4"
+tomli = "^2.0.1"
+tomli-w = "^1.0.0"
```

Comment:
Why the introduction of these two over the `toml` dependency?

Reply:
My motivation was: let's not use two libs for the same thing. And this one seems to be the preferred one in the community.

 * We had this in the dependencies anyway (just not directly). 
 * Seems that tomli is the new rage, even mentioned in the [official Python docs](https://docs.python.org/3/library/tomllib.html) (well, `tomli-w` is).